src/Workspaces/CoreTestUtilities/MEF/UseExportProviderAttribute.cs

```diff
@@ -65,7 +65,6 @@ static UseExportProviderAttribute()
         public override void Before(MethodInfo? methodUnderTest)
         {
             // Need to clear cached MefHostServices between test runs.
-            MSBuildMefHostServices.TestAccessor.ClearCachedServices();
```

Comment:
don't love that this went away.  if we add mef stuf in the future, won't this be necessary?  should we keep the stubs for this around?

i'd feel more comfortable if we got rid of all the mef stuff entirely.

Reply:
All the MEF stuff is gone except the public API which is now a no-op. If it needs to come back we can do so. (And frankly I was quite surprised this also existed, which seemed like a bit of a hack in this test code to even have to be aware of this.)